app/javascript/components/CreateWork.jsx

```diff
@@ -12,22 +14,66 @@ class CreateWork extends React.Component {
       work: {
         title: '',
         material: '',
-        medium: 0,
-        availability: 0,
+        medium: "painting",
+        availability: "active",
         images: [],
+        featured_image: null,
         artist_id: this.props.artist_id
+      },
+      categories: {
+        "medium": 0,
+        "availability": 0
       }
     }
 
     this.handleChange = this.handleChange.bind(this);
```

Comment:
Remove all .bind's since we can replace this with ES6 syntax. in function definition, you can write `handleChange = (event) => {...` to avoid binding

Reply:
fixed!